### PR TITLE
Activity Detail Component with Styles in new Style folder. Add routin…

### PR DESCRIPTION
**Pull request by andrespereyra-it**
### Summary

Activity detail view made with Card component from Material-ui.
File with styles for the Card.
Routing added in App.js file.

**Link to ticket related**: https://alkemy-labs.atlassian.net/browse/OT91-50

### Evidence
Activity Detail

### CODE REVIEW

File: src/Components/Activities/Detail/Detail.js
Comment (saralavanchy) [inline src/Components/Activities/Detail/Detail.js:9]: Take care because this component do not exists on Dev yet. It will exists after the PR is merged

File: src/Components/Activities/Detail/Detail.js
Comment (saralavanchy) [inline src/Components/Activities/Detail/Detail.js]: this space is unnecessary

File: src/Components/Activities/Detail/Detail.js
Comment (saralavanchy) [inline src/Components/Activities/Detail/Detail.js]: this space is unnecessary

File: src/Components/Activities/Detail/Detail.js
Comment (saralavanchy) [inline src/Components/Activities/Detail/Detail.js]: this space is not necessary

File: src/Components/Activities/Detail/Detail.js
Comment (saralavanchy) [inline src/Components/Activities/Detail/Detail.js]: this space is unnecessary

File: src/Components/Activities/Detail/Detail.js
Comment (saralavanchy) [inline src/Components/Activities/Detail/Detail.js]: why are you testing typeof undefined and not use directly something like ``` activity.description && setActivityDesc(...) ``` ?

File: src/Components/Activities/Detail/Detail.js
Comment (saralavanchy) [inline src/Components/Activities/Detail/Detail.js]: is this a list of activities in desc order? is this a description? The name is not clear enought, maybe something like activityDescription (if that is what it is) its a better name.

File: src/Components/Activities/Detail/Detail.js
Comment (saralavanchy) [inline src/Components/Activities/Detail/Detail.js]: the calls to the backend on the services. Please move this to the activity service.

**Review by saralavanchy — CHANGES_REQUESTED** [review]

**Comment by saralavanchy** [comment]
this user story depends on #1

File: src/Components/Activities/Detail/Detail.js
Comment (andrespereyra-it) [reply src/Components/Activities/Detail/Detail.js:9]: But It is a requirement to use the Title component. Should I wait until you merged the Title PR before send this Component to merge?

File: src/Components/Activities/Detail/Detail.js
Comment (andrespereyra-it) [reply src/Components/Activities/Detail/Detail.js]: What activity service? Sorry if I overextend but I'm not sure what you mean.
I left it this way because like the api service is not ready, I don't know how the files are going to be called. So I wrote something temporal which works. 
Accordingly to what we discuss in the #libraries discord channel, we were going to use an axios instance. I imagine something similar to this:

**Axios Instance Service file:**
```
export default axios.create({
  baseURL: "http://ongapi.alkemy.org/api/",
  headers: {
    "Content-type": "application/json",
  },
});
```

**HTTP Actions Service file:**
```
const get = (id) => {
  return axiosInstance.get(`/activities/${id}`);
};
```

So, you mean I should replace the axios call in Details.js for something like this?

`import getActivity from "../../../Services/publicApiService"`

`const res = await getActivity.get(id);`

Comment (saralavanchy) [reply src/Components/Activities/Detail/Detail.js]: lo escribo en españal, siempre que uses business logic tiene que ir en un archivo service. Si el archivo no existe entonces tenes que crearlo, en este caso en services tenes que crear un archivo userService donde se maneje toda la business logic de los users, como get con axios, etc.

Comment (saralavanchy) [reply src/Components/Activities/Detail/Detail.js]: o en un activity service depende que grupo de endpoits estas llamando. En este caso la function se llama getUsers pero estas llamando http://ongapi.alkemy.org/api/activities/${id} por lo que el codigo no es claro

Comment (saralavanchy) [reply src/Components/Activities/Detail/Detail.js]: ademas, la function esta dentro del useEffect, en el useEffect solo se deberian llamar functions

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Components/Activities/Detail/Detail.js
Comment (andrespereyra-it) [reply src/Components/Activities/Detail/Detail.js]: Done

**Review by saralavanchy — APPROVED** [review]

**Pull request merged**
